homeworks/Andrii.Morozov_AndriiMorozov88/6-typesript/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+
+module.exports = {
+    entry: './src/script_typescript.ts',
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
```

Comment:
```suggestion
                test: /\.ts$/,
```
Тут також не потрібна підтримка tsx файліф